ci_images/#bionic_builder.dockerfile#

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:bionic
```

Comment:
I think `ci/images/#bionic_builder.dockerfile#` is added by mistake.

Reply:
Hmmm.. I don't see it https://github.com/kiwix/kiwix-build/tree/new-ci-container-build/.github/workflows. But it would be an error

Reply:
This is in `ci_images` directory